ntc_templates/templates/fsas_sir_show_system_information.textfsm

```diff
@@ -0,0 +1,29 @@
+Value VERSION (\d\d.\d\d)
```

Comment:
Simplify the numeric part of the VERSION capture group.
Escape the period since we want a literal period.

```suggestion
Value VERSION (\d+\.\d+)
```